ps/jenkins/percona-server-for-mysql-8.0.groovy

```diff
@@ -574,11 +574,31 @@ parameters {
             slackNotify("${SLACKNOTIFY}", "#00FF00", "[${JOB_NAME}]: build has been finished successfully for ${BRANCH} - [${BUILD_URL}]")
             slackNotify("${SLACKNOTIFY}", "#00FF00", "[${JOB_NAME}]: Triggering Builds for Package Testing for ${BRANCH} - [${BUILD_URL}]")
             unstash 'properties'
+            currentBuild.description = "Built on ${BRANCH}; path to packages: ${COMPONENT}/${AWS_STASH_PATH}"
+            REVISION = sh(returnStdout: true, script: "grep REVISION test/percona-server-8.0.properties | awk -F '=' '{ print\$2 }'").trim()
+            PS_RELEASE=$(echo ${BRANCH} | sed 's/release-//g')
+            echo "${REVISION} is the commit ID"
+
             script {
-                currentBuild.description = "Built on ${BRANCH}; path to packages: ${COMPONENT}/${AWS_STASH_PATH}"
-                REVISION = sh(returnStdout: true, script: "grep REVISION test/percona-server-8.0.properties | awk -F '=' '{ print\$2 }'").trim()
-                echo "${REVISION} is the commit ID"
-                build job: 'package-testing-ps80', propagate: false, wait: false, parameters: [string(name: 'product_to_test', value: 'ps80'),string(name: 'install_repo', value: "testing"),string(name: 'node_to_test', value: "all"),string(name: 'action_to_test', value: "all"),string(name: 'check_warnings', value: "yes"),string(name: 'install_mysql_shell', value: "no"),string(name: 'REVISION', value: "${REVISION}")]
+                withCredentials([string(credentialsId: 'PXC_GITHUB_API_TOKEN', variable: 'TOKEN')]) {
+                sh """
+                    set -x
+                    git clone https://jenkins-pxc-cd:$TOKEN@github.com/Percona-QA/package-testing.git
+                    cd package-testing
+                    git config user.name "jenkins-pxc-cd"
+                    git config user.email "it+jenkins-pxc-cd@percona.com"
+                    cat .git/config
+                    OLD_REV=$(cat VERSIONS | grep PS80_REV | cut -d '=' -f2- | sed 's/"//g')
+                    OLD_VER=$(cat VERSIONS | grep PS80_VER | cut -d '=' -f2- | sed 's/"//g')
+                    sed -i "s/$OLD_REV/${REVISION}/g" VERSIONS
+                    sed -i "s/$OLD_VER/${PS_RELEASE}/g" VERSIONS
+                    git diff
+                    git add -A
+                    git commit -m "add ${REVISION} and ${PS_RELEASE} for ps80 package testing VERSIONS file."
```

Comment:
Please add to comment smth like Autocommit, to see that it was committed by task.